### PR TITLE
Messages, Checks, admin panel

### PR DESCRIPTION
**Pull request by UjjwalAggarwal-1**

### CODE REVIEW

File: sinimg/templates/sinimg/upload.html [hunk 1]
```diff
@@ -1,5 +1,9 @@
 {% extends 'base.html' %} {% block body %}
-
+{% if messages %}
```
Comment (kailashchoudhary11) [inline sinimg/templates/sinimg/upload.html]: You can place this inside templates/base.html instead!

**Review by kailashchoudhary11 — CHANGES_REQUESTED** [review]
Thank you. Please make these changes! Everything else seems good to me

File: sinimg/templates/sinimg/upload.html (hunk 1 above)
Comment (UjjwalAggarwal-1) [reply sinimg/templates/sinimg/upload.html]: done

**Comment by UjjwalAggarwal-1** [comment]
I see, its more subtle that way

**Review by kailashchoudhary11 — APPROVED** [review]
LGTM :) Thank you! Don't forget to star the repo!

**Comment by kailashchoudhary11** [comment]
@UjjwalAggarwal-1 Let me know if you would like to contribute more!

**Pull request merged**